### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by mom1**
- Improve README.md
- wraps for paginated decorator

This is needed for api-client-pydantic to work.

After the release of these changes, I can make the first release of the package [api-client-pydantic](https://github.com/mom1/api-client-pydantic)

### CODE REVIEW

**Review by MikeWooster — CHANGES_REQUESTED** [review]
just a small change for the readme.

File: README.md [hunk 1]
```diff
@@ -20,17 +20,18 @@ that code away from the clean abstraction you have designed.
 8. [Handling bad requests and responses](#Exceptions)
 9. [Endpoints as code](#Endpoints)
 10. [Marshalling requests/responses](#Marshalling)
+11. [Pydantic requests/responses](#Pydantic)
```
[the remark this answers is not in the document]
Comment (mom1) [reply README.md:23]: Do I need to return this section?

[the remark this answers is not in the document]
Comment (MikeWooster) [reply README.md:23]: there's no Pydantic section in the readme.  Probably best to remove this, or change it to an `Extensions` section where we can put the jsonmarshal/pydantic stuff.

**Review by MikeWooster — APPROVED** [review]

**Comment by MikeWooster** [comment]
would you mind fixing up your commits into a single commit for me before this is merged?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mom1) [reply README.md]: Done

**Comment by mom1** [comment]
> would you mind fixing up your commits into a single commit for me before this is merged?

Done

**Review by MikeWooster — APPROVED** [review]

**Pull request merged**